### PR TITLE
Fixes #362 - Implemented PR352: One-Pole Filter Audio Worklet, Updated Noise-Generator, Volume-Meter, and Audio-Worklet-Node-Options

### PR DESCRIPTION
**Pull request by tarunsinghofficial**
Fixes #362 implemented the PR352 - _one-pole-filter_ audio worklet sample.

### CODE REVIEW

**Review by hoch — APPROVED** [review]
LGTM

Thanks for your contribution!

**Comment by tarunsinghofficial** [comment]
Hi @hoch 

All 4 examples are now completed. It's now ready to merge.

**Comment by duojet2ez** [comment]
We discussed having separate pull requests for each example rather than completing all four in a single PR.

**Comment by tarunsinghofficial** [comment]
> We discussed having separate pull requests for each example rather than completing all four in a single PR.

So, shall I create separate PRs for each task?

**Comment by hoch** [comment]
Thanks for sharing your thoughts, @duojet2ez!

I think the scope of this PR is adequate - now we found a common pattern that works reasonably well for all examples, so the review was also fast.

**Review by hoch — APPROVED** [review]

**Comment by hoch** [comment]
There's a check failing and I'll have to take a look at what's going on.

**Comment by hoch** [comment]
@tarunsinghofficial Could you please update the title of PR to reflect the increased scope?

**Comment by tarunsinghofficial** [comment]
> @tarunsinghofficial Could you please update the title of PR to reflect the increased scope?

Sure, Will update the name.

File: src/audio-worklet/basic/one-pole-filter/main.js [hunk 1]
```diff
@@ -3,31 +3,55 @@
...
-  oscillator.connect(filter).connect(context.destination);
-  oscillator.start();
+  oscillatorNode.connect(filterNode).connect(context.destination);
+  oscillatorNode.start();
 
   frequencyParam
-      .setValueAtTime(0.01, 0)
-      .exponentialRampToValueAtTime(context.sampleRate * 0.5, 4.0)
-      .exponentialRampToValueAtTime(0.01, 8.0);
+    .setValueAtTime(0.01, 0)
```
Comment (hoch) [inline src/audio-worklet/basic/one-pole-filter/main.js:21]: nit: 4-space indentation

**Review by hoch — APPROVED** [review]
LGTM with a nit. Could you fix them before we merge this?

File: src/audio-worklet/basic/one-pole-filter/main.js (hunk 1 above)
Comment (tarunsinghofficial) [reply src/audio-worklet/basic/one-pole-filter/main.js]: Added the indentation.

**Pull request merged**